executor/builder.go

```diff
@@ -1029,8 +1029,8 @@ func (b *executorBuilder) buildProjBelowAgg(aggFuncs []*aggregation.AggFuncDesc,
 	}
 
 	return &ProjectionExec{
-		baseExecutor:  newBaseExecutor(b.ctx, expression.NewSchema(projSchemaCols...), projFromID, src),
-		numWorkers:    b.ctx.GetSessionVars().ProjectionConcurrency,
+		baseExecutor: newBaseExecutor(b.ctx, expression.NewSchema(projSchemaCols...), projFromID, src),
+		//numWorkers:    b.ctx.GetSessionVars().ProjectionConcurrency,
```

Comment:
Some UT failed because of this?

Reply:
Yes, this would cause goroutine leak in `TestAggregation`.

Reply:
caused by this issue: https://github.com/pingcap/tidb/issues/8787